From 20698d3fd425a07148df916b8b1a9eada9d0b5b3 Mon Sep 17 00:00:00 2001
From: Konstantin Porotchkin <kostap@marvell.com>
Date: Sun, 21 Mar 2021 13:11:15 +0200
Subject: [PATCH 1351/1921] arm64: dts: add pin control node to cp11x dtsi file

The CP11x DTSI file includes references to non-existing pinctrl
node. This patch adds an empty pinctrl node to resolve this
reference.
The compatibility string for pinctrl node is not defined in the
common DTSI file since it depends on specific SOC and defined
in SOC-specific DTS files.

Signed-off-by: Konstantin Porotchkin <kostap@marvell.com>
Change-Id: Id6bde828cbd8ba22d5f60c373e047b4cf8d6e217
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/kernel/linux/+/48230
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Stefan Chulski <stefanc@marvell.com>
[WK: The original patch got from Marvell sdk11.21.09]
Signed-off-by: Wenlin Kang <wenlin.kang@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-cp11x.dtsi | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm64/boot/dts/marvell/armada-cp11x.dtsi b/arch/arm64/boot/dts/marvell/armada-cp11x.dtsi
index deb549f419b8..3fe54e916dac 100644
--- a/arch/arm64/boot/dts/marvell/armada-cp11x.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-cp11x.dtsi
@@ -234,6 +234,9 @@
 				#clock-cells = <2>;
 			};
 
+			CP11X_LABEL(pinctrl): pinctrl {
+			};
+
 			CP11X_LABEL(gpio1): gpio@100 {
 				compatible = "marvell,armada-8k-gpio";
 				offset = <0x100>;
-- 
2.31.1

